build/builder.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -eu
+
+image="cockroachdb/builder"
+
+function init() {
```

Comment:
i know this is a copy from the main repo, but there's some cruft here, since the dockerfile is not checked in :/